src/remotecall.jl

```diff
@@ -274,7 +274,7 @@ end
 const any_gc_flag = Threads.Condition()
 function start_gc_msgs_task()
     errormonitor(
-        Threads.@spawn begin
+        @async begin
```

Comment:
This change fixed the race. It seems to have always been a `@spawn` hence why it wasn't initially converted to `@async`.